### PR TITLE
Add replies to activity center

### LINKED ISSUE #2611: Add replies to activity center

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #2611 

Do not merge until the status-go PR is (https://github.com/status-im/status-go/pull/2257)

### CODE REVIEW

File: ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml [hunk 1]
```diff
@@ -111,16 +114,33 @@ Popup {
 
                     property int idx: DelegateModel.itemsIndex
 
+                    Component.onCompleted: {
+                        switch (model.notificationType) {
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml:118]: @emizzle mentioned before that there's no guarantee that models could be in a state where data was already available for use IIRC. Could this happen here?

File: ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml [hunk 2]
```diff
@@ -111,16 +114,33 @@ Popup {
...
+                        case Constants.acitivtyCenterNotificationTypeMention:
+                            if (!hasMentions) {
+                                hasMentions = true
+                            }
+                            break
+
+                        case Constants.acitivtyCenterNotificationTypeReply:
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml:125]: typo?

File: ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml [hunk 3]
```diff
@@ -111,16 +114,33 @@ Popup {
 
                     property int idx: DelegateModel.itemsIndex
 
+                    Component.onCompleted: {
+                        switch (model.notificationType) {
+                        case Constants.acitivtyCenterNotificationTypeMention:
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml:119]: typo?

**Review by richard-ramos — APPROVED** [review]
Nice!

File: ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml (hunk 1 above)
Comment (jrainville) [reply ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml:121]: I don't think so, because the List will not be rendered unless there is data

**Pull request merged**